opencv/src/OpenCV/Internal/C/FinalizerTH.hs

```diff
@@ -38,6 +38,16 @@ Generated code (stylized):
 > }
 
 > foreign import ccall "&deleteFoo" deleteFoo :: FunPtr (Ptr Foo -> IO ())
+
+And
+
+> mkFinalizer DeletePtr "deleteFoo" "CFoo" ''Foo
```

Comment:
`DeletePtr` should be `ReleaseDeletePtr` here.

Reply:
Oh right, done.